packages/opencensus-exporter-stackdriver/src/stackdriver.ts

```diff
@@ -133,7 +133,7 @@ export class StackdriverTraceExporter implements types.Exporter {
    * authenticates the client and calls a method to send the traces data.
    * @param stackdriverTraces
    */
-  private authorize(stackdriverTraces) {
+  private authorize(stackdriverTraces: {}) {
```

Comment:
Please use more comprehensive types here.

Reply:
Ok, just created types to address that.